Environment/Environment.pillar

```diff
@@ -806,24 +764,22 @@ right-clicking on the stack frames provide equivalent functionality.
 !!Finding methods
 @sec:methodFinder
 
-There are two tools in Pharo to help you find messages. They differ in both
-interface and functionality.
+There is one tool in Pharo to help you find messages. 
```

Comment:
This part really need to be review so i'll put that into the issue for the finder.